stdlib/2.7/os/__init__.pyi

```diff
@@ -134,7 +134,16 @@ def rmdir(path: unicode) -> None: ...
 # TODO(MichalPokorny)
 def stat(path: unicode) -> Any: ...
 
-# TODO: stat_float_times, statvfs, tempnam, tmpnam, TMP_MAX, walk
+# TODO: stat_float_times, statvfs, tempnam, tmpnam, TMP_MAX
+def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
+         followlinks: bool = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
+                                                      List[AnyStr]]]: ...
```

Comment:
You can currently only share code when the two PY versions use an *identical* stub -- then it can go in stdlib/2and3 or thirdparty/2and3. In the future we'll support some version checks (as required by PEP 484) but mypy doesn't support those yet.